pro/src/ui-kit/form/MultiSelectAutoComplete/__specs__/MultiSelectAutocomplete.spec.tsx

```diff
@@ -40,217 +40,170 @@ describe('src | ui-kit | form | MutliSelectAutocomplete', () => {
     }
     const initialValues = { departement: ['01', '02'] }
 
-    it('should not display options by default', () => {
-      // given
+    it('should display field', () => {
       render(
         <Formik initialValues={initialValues} onSubmit={() => {}}>
           <MultiSelectAutocomplete {...props} />
         </Formik>
       )
-
-      // then
-      expect(screen.queryByLabelText('Ain')).not.toBeInTheDocument()
-      expect(screen.queryByLabelText('Cantal')).not.toBeInTheDocument()
+      expect(screen.getByLabelText('Département')).toBeInTheDocument()
     })
 
-    it('should display all options when the user opens the field', async () => {
-      // given
+    it('should display the number of selected options', async () => {
       render(
         <Formik initialValues={initialValues} onSubmit={() => {}}>
           <MultiSelectAutocomplete {...props} />
         </Formik>
       )
-
-      // when the user opens the field
-      await userEvent.click(
-        screen.getByRole('textbox')
-      )
-
-      // then
+      await userEvent.click(screen.getByRole('textbox'))
       expect(await screen.findAllByRole('checkbox')).toHaveLength(15)
```

Comment:
j'ai l'impression que le nom du test ne correspond pas à ce qui est testé ? `should display the number of selected options` mais on teste `expect(await screen.findAllByRole('checkbox')).toHaveLength(15)`

Reply:
bien vu, merci ! c'est corrigé